v1.1.0 - Extract plugin ceremony into automatic convenience methods on Team instances (#20)
rubyatscale/code_teams: README.md

1) You can now use the following API to get GitHub information about that team:
```ruby
team = CodeTeams.find('My Team')
members = team.github.members
github_name = team.github.team
```

>    * Or if your accessor method name differs from your plugin's class name:

Reviewer: ```suggestion
   Alternatively, you can assign an accessor method name differs from the plugin's class name:
```

   ```ruby
   class MyPlugin < CodeTeams::Plugin
     data_accessor_name :other_name

     def other_name
       # ...
     end
   end

>    # You can then use:
>    team.other_name
>    # similarly to the Github example above

Reviewer: ```suggestion
   # You can then access data in the following manner:
   team.other_name.attribute_name
```

   ```

   However, to avoid confusion, it's recommended to use the naming convention 
   whenever possible so that your accessor name matches your plugin's name

> 

Reviewer: **suggestion**: remove the `*` bullet point and align the indentation

the way the markdown is written (appears in the original too), the code block will not only mess with the continuation of the ordered list (mentioned in previous comment), but will also mess a bit with the child unordered list item as well. Namely, I think it might not render as a child...

<img width="1432" alt="Screenshot 2025-05-28 at 4 58 32 PM" src="https://github.com/user-attachments/assets/f612bde2-a85b-4c48-9014-f87d5b12691a" />

fixing only the indentation of the first list...

<img width="1438" alt="Screenshot 2025-05-28 at 4 59 11 PM" src="https://github.com/user-attachments/assets/55422c13-ae0a-423c-8c36-eff48ff6d285" />

or if the `*` is removed and everything aligned it should render like this
<img width="1472" alt="Screenshot 2025-05-28 at 4 59 58 PM" src="https://github.com/user-attachments/assets/7b8b498b-f854-4af4-94c9-7a0af4d7a723" />
